ludwig/datasets/mixins/download.py

```diff
@@ -163,3 +163,35 @@ def download_raw_dataset(self):
     @property
     def download_url(self):
         return self.config["download_urls"]
+
+
+class ZipandUncompressedFileDownloadMixin:
```

Comment:
pandas can load a zipped file, so you can use the `UncompressedFileDownloadMixi` without the need for this new one.